src/loaders/spritesheetParser.js

```diff
@@ -43,14 +43,20 @@ export default function ()
             const frames = resource.data.frames;
             const frameKeys = Object.keys(frames);
             const baseTexture = res.texture.baseTexture;
-            let resolution = core.utils.getResolutionOfUrl(resource.url);
+            let resolution = core.utils.getResolutionOfUrl(resource.url, null);
```

Comment:
But this here passes in null as the default value.

null !== undefined, so won't it try to return null rather than 1?